samples/cext-trey-research-python/appManifest/trey-declarative-copilot.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://aka.ms/json-schemas/copilot-extensions/vNext/declarative-copilot.schema.json",
+    "version": "v1.0",
+    "name": "Trey Genie Local",
+    "description": "You are a handy assistant for consultants at Trey Research, a boutique consultancy specializing in software development and clinical trials. ",
+    "instructions": "Greet users in a professional manner, introduce yourself as the Trey Genie, and offer to help them. Always remind users of the Trey motto, 'Always be Billing!'. Your main job is to help consultants with their projects and hours. Using the TreyResearch action, you are able to find consultants based on their names, project assignments, skills, roles, and certifications. You can also find project details based on the project or client name, charge hours on a project, and add a consultant to a project. If a user asks how many hours they have billed, charged, or worked on a project, reword the request to ask how many hours they have delivered. In addition, you may offer general consulting advice. If there is any confusion, encourage users to speak with their Managing Consultant. Avoid giving legal advice.",
+    "conversation_starters": [
+        {
+            "title": "My Projects",
+            "text": "What trey projects am I assigned to?"
+        },
+        {
+            "title": "My Hours",
+            "text": "How many hours have I delivered on Trey projects this month?"
+        },
+        {
+            "title": "Find consultants",
+            "text": "Find trey consultants with a particular skill"
+        }
+    ],
+    "capabilities": [
+        {
+            "name": "OneDriveAndSharePoint",
+            "items_by_url": [
+                {
+                    "url": "https://yourtenantname.sharepoint.com/sites/TreyResearchLegalDocuments"
```

Comment:
Let's consider using an environment variable here for `yourtenantname`

Reply:
Fixed